.gitlab/source_test/ebpf.yml

```diff
@@ -1,8 +1,12 @@
 ---
 .retrieve_sysprobe_deps:
-  - $S3_CP_CMD $S3_PERMANENT_ARTIFACTS_URI/clang-$ARCH-11.0.1.tar.xz /tmp/clang.tar.xz
-  - mkdir -p $DATADOG_AGENT_EMBEDDED_PATH
-  - tar -xvf /tmp/clang.tar.xz -C $DATADOG_AGENT_EMBEDDED_PATH
+  - mkdir -p $DATADOG_AGENT_EMBEDDED_PATH/bin
+  - mkdir -p $DATADOG_AGENT_EMBEDDED_PATH/include
+  - $S3_CP_CMD $S3_PERMANENT_ARTIFACTS_URI/clang.$ARCH $DATADOG_AGENT_EMBEDDED_PATH/bin/clang
+  - $S3_CP_CMD $S3_PERMANENT_ARTIFACTS_URI/clang-11.$ARCH $DATADOG_AGENT_EMBEDDED_PATH/bin/clang-11
```

Comment:
do we really need to have a versioned version ? clang-<major> here ?
or clang call itself clang-<major> ?

Reply:
I agree that we only need a single `clang` file, so renaming the binary to `clang` when copying from S3 during the package builds makes the most sense.